.github/workflows/docker.yml

```diff
@@ -24,16 +24,14 @@ on:
       - .github/workflows/docker.yml
     branches:
       - master
-    tags:
-      - v[12].[0-9]+.[0-9]+
 
 env:
   IMAGE_NAME: octodns-sync
 
 jobs:
   # Prove the image successfully builds.
   # See also https://docs.docker.com/docker-hub/builds/automated-testing/
-  test:
+  test-build:
```

Comment:
This change required updating the repository's branch protection rules, which had required `test` to pass.